refinery/ui/source/js/user-file-browser/ctrls/user-file-browser-files-ctrl.js

```diff
@@ -74,6 +82,8 @@
           // TODO: Should there be something that wraps up this "then"? It is repeated.
           // gridOptionsService.columnDefs = userFileBrowserFactory.createColumnDefs();
           gridOptionsService.data = userFileBrowserFactory.createData(solr.nodes);
+          userFileBrowserFactory.dataSetNodes.nodesCount = solr.nodes.length;
+          userFileBrowserFactory.dataSetNodes.nodesCount = solr.nodes_count;
```

Comment:
Reassignment to `userFileBrowserFactory.dataSetNodes.nodesCount`?

Reply:
typo, thanks

Reply:
Typo. 

@scottx611x The data updates are triggered after different events, so no. The whole thing could use refactoring, but given it's a end-of-the milestone fix, I'd let it pass.